packages/desktop-client/src/components/App.tsx

```diff
@@ -140,9 +140,17 @@ function AppWrapper() {
       }
     }
 
+    function onVisibilityChange() {
+      sync();
+    }
+
     window.addEventListener('focus', checkScrollbars);
+    window.addEventListener('visibilitychange', onVisibilityChange);
 
-    return () => window.removeEventListener('focus', checkScrollbars);
+    return () => {
+      window.removeEventListener('focus', checkScrollbars);
+      window.removeEventListener('visibilitychange', onVisibilityChange);
+    };
   }, []);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I believe we should also add `sync` to the dependencies here now.

```suggestion
  }, [sync]);
```

Reply:
Agreed, done via https://github.com/actualbudget/actual/pull/1549/commits/765674878abe824974f9c21f910991f8b0c6ae8c